data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -136,4 +139,8 @@ public class BeaconRestApiTypes {
           BEACON_BLOCK_ROOT,
           BYTES32_TYPE.withDescription(
               "`bytes32` The block root for which to produce the contribution."));
+
+  public static final ParameterMetadata<Long> TARGET_PEER_COUNT_PARAMETER =
+      new ParameterMetadata<>(
+          TARGET_PEER_COUNT, LONG_TYPE.withDescription(TARGET_PEER_COUNT_DESCRIPTION));
```

Comment:
we're basically limited to int for a bunch of reasons, so this could use integer_string...